ext/opentelemetry-ext-dbapi/src/opentelemetry/ext/dbapi/__init__.py

```diff
@@ -103,7 +103,7 @@ def wrap_connect(
     """
 
     # pylint: disable=unused-argument
-    def wrap_connect_(
+    def _wrap_connect(
```

Comment:
Nit: this change is not necessary. Adding an underscore at the end is a very normal way of naming symbols whose main name is already taken. On the other hand adding an underscore at the beginning actually changes the meaning of the symbol, indicating it is private in some context.

Reply:
It's easier to differentiate between `wrap_connect` and `_wrap_connect` than `wrap_connect` and `wrap_connect_`. It can be private in this case too.